tests/test_figurewidget_resampler.py

```diff
@@ -794,7 +794,7 @@ def test_hf_data_property_reload_data():
     assert np.all(fwr.hf_data[0]["y"] == new_y)
 
     fwr.reload_data()
-    assert (fwr.data[0]["x"][0] >= 10_000) & (fwr.data[0]["x"][-1] <= 20_000)
+    assert (fwr.data[0]["x"][0] >= 9_999) & (fwr.data[0]["x"][-1] <= 20_001)
```

Comment:
I would indeed suspsect that, like here, you need to alter both boundaries.

Reply:
thnx!